.gitignore

```diff
@@ -0,0 +1,7 @@
+/.idea
+/out
+/target
+*.iml
+*.log
+*.img
+*.png
```

Comment:
Please remove *.png because we need the png files to compare logos if they were changed due to some regression

Reply:
Done